src/containers/ViewerPanel/index.tsx

```diff
@@ -431,6 +462,62 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
         }
     };
 
+    private updateDisplayBreakpoints = () => {
+        if (location.pathname === EMBED_PATHNAME) {
+            this.setState({
+                embedDisplayBreakpoints: {
+                    belowControlsHeight:
+                        window.innerHeight <=
+                        EMBEDDED_STANDARD_CONTROLS_MINIMUM_HEIGHT,
+                    belowControlsWidth:
+                        window.innerWidth <=
+                        EMBEDDED_STANDARD_CONTROLS_MINIMUM_WIDTH,
+                    belowScaleBarWidth:
+                        window.innerWidth <= EMBEDDED_SCALE_BAR_MINIMUM_WIDTH,
+                },
+            });
+        }
+    };
+
+    private handleFullscreenChange = () => {
+        const { setEmbedFullscreen } = this.props;
+        setEmbedFullscreen(!!document.fullscreenElement);
+        const current = this.centerContent.current;
+        if (current) {
+            setTimeout(() => {
+                this.resize(current);
+                this.updateDisplayBreakpoints();
+            }, 100);
+        }
+    };
+
+    private toggleFullscreen = () => {
+        const { embedFullscreen } = this.props;
+        if (embedFullscreen) {
+            document.exitFullscreen();
+        } else {
+            document.documentElement.requestFullscreen().catch((err) => {
+                console.error(
+                    `Error attempting to enable fullscreen: ${err.message}`
+                );
+            });
+        }
+    };
+
+    private get embedDisplaySettings() {
```

Comment:
if I'm reading this right, this will happen regardless of if it's embedded or not? ie, if they're looking at it on a phone, but it seems like this name isn't correct then